client/src/data/auth.ts

```diff
@@ -20,7 +20,7 @@ export const login = async (data: AuthBody): Promise<UserNoAccessToken> => {
     .then((res) => res.data);
   // Set data in localStorage
   set(ACCESS_TOKEN_KEY, accessToken);
-  set(ACCESS_TOKEN_EXPIRY_KEY, `${getExpiryOfJwt(accessToken)}`);
+  set(ACCESS_TOKEN_EXPIRY_KEY, `${getExpiryOfJwt(accessToken)}000`);
```

Comment:
why are the `000` needed

Reply:
will add comment, but its because js Date object is in milliseconds, whereas jwt date is in seconds.

the bug was because when we transform the jwt expiry into a date obj, it'll like never hit the condition to logout

Reply:
I think we should use `date-fns`'s `parseJSON` to parse the JSON date string instead of manually adding the three 0s and use date-fns to perform the comparison

What do you think?

Reply:
oh cool didn't know about that. leaving this here for posterity

https://github.com/date-fns/date-fns/blob/main/src/parseJSON/index.ts

Reply:
ok nevermind I looked into it and it won't work.

If you want it to be more explicit in naming the closest thing is this:

```
fromUnixTime(+getExpiryOfJwt(accessToken)).getTime().toString()
```

but at that point I would would rather have the original:
```
`${getExpiryOfJwt(accessToken)}000`);
```

lmk which one you prefer. I personally want to stick with the original one.

Reply:
Just leave the comment referencing this discussion and we should be gtg